core/templates/dev/head/pages/admin-page/admin-page.directive.ts

```diff
@@ -102,6 +105,12 @@ oppia.directive('adminPage', ['UrlInterpolationService',
           ctrl.isRolesTabOpen = AdminRouterService.isRolesTabOpen;
           ctrl.isMiscTabOpen = AdminRouterService.isMiscTabOpen;
 
+          $http.get('/csrfhandler').then(function(response) {
+            // This is done outside the service to prevent
+            // circular dependency issues.
+            CsrfTokenService.setToken(response.data.token);
+          });
```

Comment:
Since we use jQuery now can we also call the service here?

Reply:
I think this should be removed altogether since the call will be made by App.ts